packages/common/src/store/pages/profile/actions.ts

```diff
@@ -150,6 +153,11 @@ export type SetNotificationSubscriptionAction = {
   handle?: string
 }
 
+export type SetIsInitialFetchAfterSSRAction = {
```

Comment:
guessing this is a pattern we did for track page too? the idea is so client side we know if weve already requested data from server, meaning we should rerequest for user-specific data?

Reply:
yea it places any virtual fields that we add onto the cached entities also (like _cover_art_sizes so that the images work properly)

Reply:
this also seems like a prime candidate for something that updated to be in one place bc i would hate to have to write this for every page

Reply:
Yeah we use this to replace the cache entry when the personalized version of the data comes back. Could remove this if we had endpoints specifically for fetching the personalized fields

Reply:
yeah fwiw we have this notion of a "local" file in the cache, which tells the cache to always override it. maybe we can have a similar thing, and then pages check that value to determine if it should refetch, like `personalized` field or something